src/App.tsx

```diff
@@ -1,17 +1,121 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
-
+import { TodoList } from './components/TodoList';
+import { PreparedTodo, User } from './app.typedefs';
+import todos from './api/todos';
 import users from './api/users';
 
+const findUser = (userId: number):User => {
+  const selectedUser = users.find(user => user.id === userId);
+
+  const name = selectedUser?.name;
+  const email = selectedUser?.email;
+
+  return { name, email } as User;
```

Comment:
nope. what if the user actually does not exist? your app will crash. try to never use `as`

Reply:
Still need to fix that. Don't use type assertion unless completely necessary. In this case it is not.